test/checkout.int.t.sol

```diff
@@ -962,26 +964,28 @@ contract Checkout_buyingWithAlternateERC20_floatingPointPriceMultiple is
   }
 }
 
-contract Checkout_buyingWithCustomFee is Checkout {
+contract Checkout_swapWithoutFeeSpecialOrder is Checkout {
   uint256 ownerPrivateKey = 0xA11CE;
   address owner = vm.addr(ownerPrivateKey);
   uint256 customFee = 5;
   uint256 certificateAmount = 1 ether;
+  uint256 customPriceMultiple = 1800; // $18.00 -- test below the default price multiple of $20.00
```

Comment:
made sure one test with a custom price was deviating from default price multiple with a lower price, and the other test deviates higher